_episodes/requirements.md

```diff
@@ -18,4 +18,91 @@ By the end of this episode,
 learners should be able to
 __describe all the requirements for running a tool__
 
+
+> ## Exercise 1:
+>
+> For each of these variables, is it required in a CommandLineTool?
```

Comment:
```suggestion
> For each of these variables, is it required in a `CommandLineTool`?
```

I recommened to use backticks like this whenever referring to text in the CWL specification: it will make the text much more readable e.g. to tell the difference between talking about `inputs`, the field in the CWL description file, and inputs more generally.